test/case/ietf_routing/ospf_default_route_advertise/test.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
+"""
+OSPF Default route advertise
+
+Test advertising default route adverting, sometimes called 'redistribute origin' both
+always (regardless if a local default route exist or not) and also redistribute only
+when local default route exist.
+"""
+
+import infamy
+import infamy.route as route
+from infamy.util import until, parallel
+
```

Comment:
Ask pylint, it'll tell you that you need two line breaks here, like you do between methods, below.